src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,28 +1,105 @@
+import random
 # TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+def insertion_sort( arr ):
     # loop through n-1 elements
-    for i in range(0, len(arr) - 1):
+    for i in range(0, len(arr)):
```

Comment:
Great job figuring out that  For loop in Python is not inclusive of the upper bound.